README.md

```diff
@@ -63,7 +63,7 @@ If you have multiple Python versions installed, you can identify which Python
 version `node-gyp` uses by setting the `--python` variable:
 
 ``` bash
-$ node-gyp --python /path/to/python2.7
+$ node-gyp configure --python /path/to/python2.7
```

Comment:
The option applies to more than just `configure`, but what you could do here is this:

---

<pre>
```
$ node-gyp &lt;command&gt; --python /path/to/python2.7
```

Where `&lt;command&gt;` is one of the [commands](#commands) listed below.
</pre>

---

 (I'm using pre tags to format this, don't use the raw markdown, use the rendered markdown, also the 'bash' type should be dropped here, it gets in the way of proper formatting)

Reply:
Also, this line uses __/path/to/python2.7__ but...
the example just a few lines below uses __/path/to/executable/python2.7__

This would be more readable it we used consistent paths in these two examples.